integration/kubeflow/rbac.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I think it's not clusterRole. It should be Role in specified namespace.

Reply:
Sure. It will be more safe in that case. For standalone KFP deployment, maybe we can only set this namespace to which KFP was deployed. [More details about the namespace in Kubeflow.](https://www.kubeflow.org/docs/components/central-dash/profiles/)